tests/conftest.py

```diff
@@ -46,6 +48,6 @@ def keycloak_config(db):
     # in case caching is setup, ensure that it is invalidated
     config.save()
 
-    yield
+    yield config
```

Comment:
Just for my understanding: why do pytest fixtures need to return a generator? :thinking:

Reply:
they don't need to, but using a generator allows you to do cleanup after the test (like clearing the cache again)

The object that is `yield`-ed is then available as the value of the fixture - it's the equivalent of `return <foo>` :)